src/Microsoft.AspNet.Razor.Runtime/Runtime/TagHelpers/TagHelperDesignTimeDescriptorFactory.cs

```diff
@@ -90,31 +93,51 @@ public static TagHelperDesignTimeDescriptor CreateDescriptor(Type type)
             return null;
         }
 
-        private static DocumentationDescriptor CreateDocumentationDescriptor(Assembly assembly, string id)
+        private XmlDocumentationProvider GetXmlDocumentationProvider(Assembly assembly)
         {
-            var assemblyLocation = assembly.Location;
+            var hashCodeCombiner = HashCodeCombiner.Start();
+            hashCodeCombiner.Add(assembly);
+            hashCodeCombiner.Add(CultureInfo.CurrentCulture);
```

Comment:
Culture can change half way, yes. There's a setting in VS that exposes itself via `CultureInfo.CurrentCulture`.